pkg/apimachinery/identity/context.go

```diff
@@ -111,6 +111,8 @@ var serviceIdentityPermissions = getWildcardPermissions(
 	"datasources:delete",
 	"alert.provisioning:write",
 	"alert.provisioning.secrets:read",
+	"users:read",    // ???
+	"users:read:id", // ???
```

Comment:
@kalleep  @chaudyg -- what values are needed to be able to list users from iam?  with these I get an empty list :(

https://github.com/grafana/grafana/blob/main/pkg/registry/apis/iam/authorizer.go#L20